src/bundle/Core/Resources/config/default_settings.yml

```
@@ -173,7 +173,7 @@ parameters:
     ibexa.site_access.config.default.image.published_images_dir: images
     ibexa.site_access.config.default.image.versioned_images_dir: images-versioned
 
-    ibexa.site_access.config.default.variation_handler_identifier: alias
+    ibexa.site_access.config.default.variation_handler_identifier: '%env(string:VARIATION_HANDLER_IDENTIFIER)%'
```

First, we should provide a default value:
```suggestion
    ibexa.site_access.config.default.variation_handler_identifier: '%env(default:"alias":string:VARIATION_HANDLER_IDENTIFIER)%'
```

Second, the environment variable should be prefixed by our vendor name:
```suggestion
    ibexa.site_access.config.default.variation_handler_identifier: '%env(default:"alias":string:IBEXA_VARIATION_HANDLER_IDENTIFIER)%'
```

It could be also more specific to indicate it's about image variation handling.

---

TBH I would see this override in recipes only, but what Steve suggested is good enough.

---

If it's a tag, then it should be named as tag. Personally I'd expect the identifier to be straight up service name, just looking at the expected environment variable name.

I'm not a fan of using environment variables in a library, and I'd prefer to be able to use bundle configuration for it. That's the Symfony's recommended way, and while we often debate about what is and what is not actually good for us, I'd say eventually this should be placed in bundle config. It makes upgrades somewhat easier IMO.

It goes out of scope atm though.